frame/identity/Cargo.toml

```diff
@@ -14,6 +14,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "0.9.0", default-features = false, features = ["derive"] }
```

Comment:
the std version of scale-info is not set when std is set. But it doesn't matter at the end because other crate should activate it.